yoti_python_sdk/tests/test_anchor.py

```diff
@@ -121,7 +122,11 @@ def test_processing_unknown_anchor_data():
     expected_timestamp = datetime(2019, 3, 5, 10, 45, 11, 840037)
     actual_timestamp = anchors[0].signed_timestamp
 
-    assert expected_timestamp == actual_timestamp
+    # due to possibility of working on different timezone
+    try:
+        assert expected_timestamp == actual_timestamp
+    except AssertionError:
+        assert expected_timestamp == actual_timestamp - timedelta(hours=1)
```

Comment:
I think we should use UTC time for input and output, rather then try in a different timezone if it fails the first time. We should have examples of this elsewhere in the codebase

Reply:
I'm waiting for code example. After trying multiple approach I do it like that - I got diffrent outputs on diffrent computers so decide to make it like that

Reply:
A few things might work:
1. does the `get_utc_offset()` function in this file help? 
2. `expiry_date.astimezone(pytz.utc)` (from https://github.com/getyoti/yoti-python-sdk/blob/1873e9adf90fc5757675931f78bca61146e663c8/yoti_python_sdk/dynamic_sharing_service/extension/third_party_attribute_extension.py#L26)
3. Creating the expected_timestamp with `, tzinfo=pytz.utc`, as in https://github.com/getyoti/yoti-python-sdk/blob/1873e9adf90fc5757675931f78bca61146e663c8/yoti_python_sdk/tests/dynamic_sharing_service/extension/test_third_party_attribute_extension.py#L99

Reply:
done